carbonserver/tests/api/test_api_black_box.py

```diff
@@ -0,0 +1,246 @@
+import os
+import random
+
+import pytest
+import requests
+
+# Get the API utl to use from an env variable if exist
+URL = os.getenv("CODECARBON_API_URL")
+if URL is None:
+    pytest.exit("CODECARBON_API_URL is not defined")
+
+experiment_id = project_id = user_id = api_key = org_id = team_id = email = None
+org_name = org_description = org_new_id = None
+team_name = team_description = team_new_id = emission_id = None
+password = "Secret1!îstring"
+
+
+def is_key_value_exist(list_of_dict, key, value):
+    """
+    Check if at least one value of a key is equal to the specified value.
+    """
+    for d in list_of_dict:
+        if d[key] == value:
+            return True
+    return False
+
+
+def is_key_all_values_equal(list_of_dict, key, value):
+    """
+    Check if all values of a key are equal to the specified value.
+    """
+    for d in list_of_dict:
+        if d[key] != value:
+            return False
+    return True
+
+
+def test_api_user_create():
+    assert URL is not None
+    email = f"test-{random.randint(1, 20_000_000)}@test.com"
```

Comment:
The random values will flood the local databases (and one day lead to a very confusing debugging session).
If you set fixed emails, you can execute delete operations from a pytest fixture, following this syntax : 
```python
from sqlalchemy import delete

USER_EMAIL = 'user@integration.test'

@pytest.fixture()
def del_user_after_test():
    """Fixture to destroy user"""
    # Setup: orm configuration

    yield # Test execution

    # Clean up user before ending test execution by pytest
    (
        delete(user_table).
        where(user_table.c.email == USER_EMAIL)
    )

def test_api_user_createt(del_user_after_test):
    """Test that creates user"""
    email = USER_EMAIL

    ...
```

Reply:
No more random and delete of created user.
I suggest to implement other delete when we have a delete route because deleting directly into database is not real blackbox test.